pages/common/ansible-vault.md

```diff
@@ -0,0 +1,28 @@
+# ansible-vault
+
+> Encrypts & decrypts values, data structures and files within Ansible projects.
+> More information: <https://docs.ansible.com/ansible/latest/user_guide/vault.html#id17>.
+
+- Create a new encrypted vault file with vault prompt for password:
+
+`ansible-vault create {{vault_file}}`
+
+- Create a new encrypted vault file with vault key file to encrypt with:
+
+`ansible-vault create --vault-password-file={{password_file}} {{vault_file}}`
+
+- Encrypt existing file, using password file (optional):
```

Comment:
How does this example differ from the previous one? Both of the commands look the same to me.

Reply:
It encrypts an existing file, unlike the other one which creates a new file aswell.

Reply:
Ah, I see. What about this then?
```suggestion
- Encrypt an existing file using an optional password file:
```